ydb/library/yql/minikql/comp_nodes/mkql_block_exists.cpp

```diff
@@ -0,0 +1,69 @@
+#include "mkql_exists.h"
+#include <ydb/library/yql/minikql/arrow/mkql_bit_utils.h>
+#include <ydb/library/yql/minikql/computation/mkql_block_impl.h>
+#include <ydb/library/yql/minikql/mkql_node_cast.h>
+
+namespace NKikimr {
+namespace NMiniKQL {
+
+namespace {
+
+class TBlockExistsExec {
+public:
+    TBlockExistsExec(const std::shared_ptr<arrow::DataType>& returnArrowType)
+        : ReturnArrowType_(returnArrowType)
+    {}
+
+    arrow::Status Exec(arrow::compute::KernelContext* ctx, const arrow::compute::ExecBatch& batch, arrow::Datum* res) const {
+        const auto& input = batch.values[0];
+        MKQL_ENSURE(input.is_array(), "Expected array");
+        const auto& arr = *input.array();
+
+        std::shared_ptr<arrow::Buffer> bitmap = ARROW_RESULT(arrow::AllocateBitmap(arr.length, ctx->memory_pool()));
```

Comment:
result is always not NULL, just pass nullptr as 0th buffer

Reply:
Rewrote this part, [introduced](https://github.com/ydb-platform/ydb/pull/3400/files#diff-d1a86e68d1bedb53992628d8e160cd0d33fd9ee0f67bed1bc5294f807a0dd178R50-R62) common helpers.